Packs/FullHunt/Integrations/FullHunt/FullHunt.py

```diff
@@ -0,0 +1,252 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+"""
+FullHunt.io API integration
+"""
+
+''' IMPORTS '''
+
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API
+    This Client implements API calls, and does not contain any Demisto logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this HelloWorld implementation, no special attributes defined
+    """
+
+    def get_account_status(self) -> str:
+        """
+        Get the account status with email, credits, etc. with the /auth/status API endpoint.
+        """
+        return self._http_request(
+            method='GET',
+            url_suffix='/auth/status',
+        )
+
+    def get_host(self, host: str) -> Dict[str, Any]:
+        """
+        Gets the host details using the '/host' API endpoint.
+
+        """
+
+        return self._http_request(
+            method='GET',
+            url_suffix=f'/host/{host}'
+        )
+
+    def get_domain_details(self, domain: str) -> Dict[str, Any]:
+        """
+        Get domain details using the '/domain/<domain>/details' API endpoint
+
+        """
+
+        return self._http_request(
+            method='GET',
+            url_suffix=f'/domain/{domain}/details'
+        )
+
+    def get_subdomain(self, domain: str) -> Dict[str, Any]:
+        """
+        Get all subdomains from a given domain using the '/domain/<domain>/subdomains' API endpoint
+
+        """
+
+        return self._http_request(
+            method='GET',
+            url_suffix=f'/domain/{domain}/subdomains'
+        )
+
+
+''' STANDALONE FUNCTION '''
+
+
+''' COMMAND FUNCTION '''
+
+
+def test_module(client: Client, params: Dict[str, Any]) -> str:
+    """
+    Tests API connectivity and authentication by using the get_account_status() function
+    """
+
+    try:
+        client.get_account_status()
+        return 'ok'
+
+    except DemistoException as e:
+        if 'Unauthorized' in str(e):
+            return 'Authorization error 401: Probably the API key is not set correctly'
+        else:
+            raise e
+
+
+def get_account_status_command(client: Client, params: Dict[str, Any]) -> CommandResults:
+    """
+    Get the information about the user and user credit
+    """
+    try:
+        response = client.get_account_status()
+        user_info = tableToMarkdown("User Info", response['user'])
```

Comment:
response is a dict. Please use response.get('user','')

Reply:
Done and pushed